DevIncubator_Autopark/VehicleType.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace DevIncubator_Autopark
+{
+	class VehicleType
+	{
+		private string _typeName;
```

Comment:
Better to use autoprops, I think describing props is not necessary in that task.